AlphaWallet/Core/Views/ButtonsBar/VerticalButtonsBar.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  VerticalButtonsBar.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 9/3/22.
+//
+
+import UIKit
+
+class VerticalButtonsBar: UIView {
+    
+    // MARK: - Public properties
+    var buttons: [UIButton] = []
```

Comment:
private before public

Reply:
> Is it because you have upgraded Xcode and I haven't? Is the upgrade necessary? And anyway this change doesn't look like it should be part of this PR, can you remove it?

@hboon Do you want me to downgrade back to 13.2.1? If so, you should announce this to the entire team. I didn't make this change, Xcode changed it by itself.

Reply:
> I didn't make this change, Xcode changed it by itself.

You should (have) check your PR for what has changed and excluded changes you didn't expect :)

> Do you want me to downgrade back to 13.2.1

Current options for you is to use the older Xcode (and use this https://gist.github.com/steipete/d9b44d8e9f341e81414e86d7ff8fb62d to make it work), install the older simulators too. Or use the newer Xcode version as long as PRs are compatible. I think the change with the SPM metadata is just a one-off incompatibility.

Also, better solution for longer term: can you help add a `.xcode-version` file #4093

Reply:
> I'm getting this in the console:
> 
> > User defaults from command line:
> > IDECustomDerivedDataLocation = /Users/hboon/Library/Caches/JetBrains/AppCode2021.2/DerivedData
> > Resolve Package Graph
> > Package.resolved file is corrupted or malformed; fix or delete the file to continue: unsupported schema version 2
> > Resolved source packages:
> > WalletConnect: https://github.com/WalletConnect/WalletConnectSwiftV2 @ 0.5.0
> > xcodebuild: error: Could not resolve package dependencies:
> > Package.resolved file is corrupted or malformed; fix or delete the file to continue: unsupported schema version 2
> 
> Is it because you have upgraded Xcode and I haven't? Is the upgrade necessary? And anyway this change doesn't look like it should be part of this PR, can you remove it?

@hboon Should we put those type of files into gitignore? It appears to be reconstructed if missing right?